tests/config.spec.ts

```diff
@@ -0,0 +1,26 @@
+import * as m from 'mochainon';
+import * as path from 'path';
+
+import config from '../lib/config';
+
+// tslint:disable no-unused-expression
```

Comment:
Personally I always just use `m.chai.expect(isAbsolute).to.equal(true)` instead, to avoid this. Whatever you prefer though is fine.